File: DataBindingStudy/app/src/main/java/com/jie/databinding/OperatorActivity.java
package com.jie.databinding;

import androidx.appcompat.app.AppCompatActivity;
import androidx.databinding.DataBindingUtil;

import android.os.Bundle;

import com.jie.databinding.databinding.ActivityOperatorBinding;
import com.jie.databinding.model.User;

public class OperatorActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityOperatorBinding operatorBinding = DataBindingUtil.setContentView(this, R.layout.activity_operator);
        User user = new User("frankie", "123456");
        operatorBinding.setUserInfo(user);
        setTitle("operator 运算符");
    }
}

File: DataBindingStudy/app/src/main/java/com/jie/databinding/ui/activity/OperatorActivity.kt
package com.jie.databinding.ui.activity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.jie.databinding.R
import com.jie.databinding.databinding.ActivityOperatorBinding
import com.jie.databinding.model.UserEntity

class OperatorActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        DataBindingUtil.setContentView<ActivityOperatorBinding>(
            this,
            R.layout.activity_operator
        ).apply {
            userInfo = UserEntity("frankie", "123456")
        }
        title = "Operator study"
    }
}